contracts/modules/royalty/RoyaltyModule.sol

```diff
@@ -201,6 +207,10 @@ contract RoyaltyModule is
         if (!$.isWhitelistedRoyaltyPolicy[payerRoyaltyPolicy])
             revert Errors.RoyaltyModule__NotWhitelistedRoyaltyPolicy();
 
+        ILicenseRegistry licenseRegistry = LICENSE_REGISTRY;
+        if (licenseRegistry.isExpiredNow(receiverIpId) || licenseRegistry.isExpiredNow(payerIpId))
```

Comment:
Why do you need to check both receiver and payer? Can u add some more comments?

Reply:
The receiver must be checked. On second thought, I agree that in case of delayed payments via ie. disputes, it could be useful to allow expired payers.

Addressed here: https://github.com/storyprotocol/protocol-core-v1/pull/91/commits/5170725b903ebaae83cd58945c474f41d8ba1861